pubtools/pulplib/_impl/client/client.py

```diff
@@ -181,10 +185,53 @@ def search_repository(self, criteria=None):
             response_f, lambda data: self._handle_page(Repository, search, data)
         )
 
+    def get_maintenance_report(self):
+        """Get the current maintenance mode status for this Pulp server.
+
+        Returns:
+            Future[:class:`~pubtools.pulplib.MaintenceReport`]
```

Comment:
Class name is misspelled.

```suggestion
            Future[:class:`~pubtools.pulplib.MaintenanceReport`]
```